src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ReactDom from "react-dom";
+import App from "./components/app/app.jsx";
+
+const Settings = {
+  ERRORS_COUNT: 3
+};
+
+ReactDom.render(
+    <App
+      errorsCount={Settings.ERRORS_COUNT}
```

Comment:
можно упростить (на случай если вам нужно будет много чего передавать как пропсы)
```suggestion
      {...settings}
```

Reply:
нет смысла при передачи перечисления